composer.json

```diff
@@ -1,70 +1,71 @@
-    },
-    "require-dev": {
-        "brain/monkey": "^2.5",
-        "codeception/codeception": "^4.1",
-        "codeception/module-asserts": "^1.3",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-        "infinum/coding-standards-wp": "0.4.3",
-        "php-stubs/wordpress-stubs": "^5.5"
-    },
-    "require": {
-        "php": ">=7.1",
-        "guzzlehttp/guzzle": "^6.0",
-        "infinum/eightshift-libs": "^2.2.2",
-        "mailchimp/marketing": "^3.0",
-        "mailerlite/mailerlite-api-v2-php-sdk": "^0.3.0",
-        "php-http/guzzle6-adapter": "^2.0",
-        "saintsystems/odata-client": "^0.4.2",
-        "szepeviktor/phpstan-wordpress": "^0.7.1"
-    },
-    "autoload": {
-        "classmap": [
-            "src/",
-            "tests/WordPressMocks/"
-        ],
-        "psr-4": {
-            "EightshiftFormsTests\\": "tests/"
-        }
-    },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "config": {
-        "sort-packages": true,
-        "optimize-autoloader": true,
-        "process-timeout": 2000
-    },
-    "scripts": {
-        "analyze": "@php ./vendor/bin/phpstan analyze",
-        "standards:check": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs",
-        "standards:fix": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf",
-        "test:all": "@php ./vendor/bin/codecept run --debug",
-        "test:unit": "@php ./vendor/bin/codecept run unit --debug"
-    }
-}
+	"name": "eightshift/eightshift-forms-plugin",
+	"description": "Eightshift Forms plugin for building developer-friendly Gutenberg-ready forms in WordPress",
+	"keywords": [
+		"composer",
+		"form",
+		"forms",
+		"email",
+		"Mailchimp",
+		"plugin",
+		"blocks",
+		"Gutenberg",
+		"WordPress"
+	],
+	"homepage": "https://eightshift.com/",
+	"license": "MIT",
+	"authors": [
+		{
+			"name": "Eightshift team",
+			"email": "team@eightshift.com",
+			"homepage": "https://eightshift.com/",
+			"role": "Developer / IT Manager"
+		}
+	],
+	"support": {
+		"issues": "https://github.com/infinum/eightshift-forms-plugin/issues",
+		"source": "https://github.com/infinum/eightshift-forms-plugin"
+	},
+	"require-dev": {
+		"brain/monkey": "^2.5",
+		"codeception/codeception": "^4.1",
+		"codeception/module-asserts": "^1.3",
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.7",
+		"infinum/eightshift-coding-standards": "^1.1",
+		"php-stubs/wordpress-stubs": "^5.5",
+		"szepeviktor/phpstan-wordpress": "^0.7"
+	},
+	"require": {
+		"php": "^7.2 || <8.0",
+		"infinum/eightshift-libs": "dev-release/3.1.0",
+		"mailchimp/marketing": "^3.0",
+		"mailerlite/mailerlite-api-v2-php-sdk": "^0.3.0",
+		"php-http/guzzle6-adapter": "^2.0",
+		"saintsystems/odata-client": "^0.4.2",
+		"szepeviktor/phpstan-wordpress": "^0.7.1"
+	},
+	"autoload": {
+		"classmap": [
+			"tests/WordPressMocks/"
+		],
+		"psr-4": {
+			"EightshiftForms\\": "src/",
+			"EightshiftFormsTests\\": "tests/"
```

Comment:
Put tests in `autoload-dev` (https://getcomposer.org/doc/04-schema.md#autoload-dev)